front/src/applications/operationalStudies/components/ImportTrainSchedule/ImportTrainScheduleConfig.tsx

```diff
@@ -40,17 +38,25 @@ export default function ImportTrainScheduleConfig({
 }: ImportTrainScheduleConfigProps) {
   const { t } = useTranslation(['operationalStudies/importTrainSchedule']);
   const [postSearch] = osrdEditoastApi.usePostSearchMutation();
```

Comment:
Should you use this format for harmonization with the rest of the app ?
```suggestion
  const [postSearch] = osrdEditoastApi.endpoints.postSearch.useMutation();
```